noxfile.py

```diff
@@ -48,6 +48,14 @@ def default(session):
     else:
         session.install("ipython")
 
+    # opentelemetry was not added to [all] because opentelemetry does not support Python 2
+    # exporter does not need to be in nox thus it has been added to README documentation
+    if session.python != "2.7":
```

Comment:
Could there be any other 2.something versions we need to avoid?

Reply:
I believe BigQuery only supports Python 2.7 (In terms of 2. versioning). Similar logic is used when installing IPython.

Reply:
That's right. And there will never be a 2.8, so this should last until BigQuery drops Python 2.